data/types/device/sensor/devicesensor_test.go

```diff
@@ -1,4 +1,4 @@
-package devicesensor_test
+package sensor_test
```

Comment:
Suggest that this file be renamed to `sensor_test.go` for consistency.

Reply:
done